### PR TITLE
Add host command line parameter to start

### PR DESCRIPTION
**Pull request by ifedotov**
Proposing simple change to add a "host" command line parameter to start script.

### CODE REVIEW

File: docs/general/commands.md [hunk 1]
```diff
@@ -66,11 +66,11 @@ npm run start:production
 Starts the production server, building the app for optimal performance: assets are
 minified and served gzipped.
 
-### Port
+### Host and Port
 
-To change the port the app is accessible at pass the `--port` option to the command
-with `--`. E.g. to make the app visible at `localhost:5000`, run the following:
-`npm start -- --port 5000`
+To change the host and/or port the app is accessible at pass the `--host` and `--port` option to the command
```
Comment (samit4me) [inline docs/general/commands.md]: Maybe a comma after "accessible at" and put another "and/or" between --host --port.

`To change the host and/or port the app is accessible at, pass the `--host` and/or `--port` option(s) to the command`

File: docs/general/commands.md [hunk 2]
```diff
@@ -66,11 +66,11 @@ npm run start:production
...
+with `--`. E.g. to make the app visible at `my-local-hostname:5000`, run the following:
+`npm start -- --port 5000 --host my-local-hostname`
```
Comment (samit4me) [inline docs/general/commands.md]: Maybe put the `--host` option before `--port` so it matches the order above.

File: server/index.js [hunk 3]
```diff
@@ -19,11 +19,14 @@ setup(app, {
...
-// get the intended port number, use port 3000 if not provided
+// get the intended host and port number, use localhost and port 3000 if not provided
+const host = argv.host || process.env.HOST || 'localhost';
 const port = argv.port || process.env.PORT || 3000;
 
+console.log(argv);
+
 // Start your app.
-app.listen(port, (err) => {
+app.listen(port, host, (err) => {
```
Comment (samit4me) [inline server/index.js]: Again a small nitpick, but for consistency maybe put the host before the port everywhere if possible.

**Comment by samit4me** [comment]
@ifedotov would you mind updating the docs and merging or rebasing the latest `dev` branch?

**Comment by ifedotov** [comment]
Yep, will do thanks.

File: server/index.js (hunk 3 above)
Comment (ifedotov) [inline server/index.js]: This is actually internal express thing (i.e app.listen() is an express call). It has host parameter after post. That's why I didn't change logger to have host before port.

Comment (samit4me) [inline server/index.js]: Yep that makes sense, thanks for clarifying.

**Pull request merged**